explainer.md

```diff
@@ -22,6 +22,13 @@ Therefore, we would like to propose a mechanism to enable security
 researchers to audit the source code of especially-sensitive web apps
 that is (or was) sent to any user, not just to themselves.
 
+```
+# jka commentary
+Rephrasing this: we'd like a way to publish the source for content distributed
+on the web, and for recipients to confirm that the content they received from
+the server matches what was expected.
+```
```

Comment:
That's fair, yep.  I think that what I see lacking is a chain-of-integrity that begins with first page-open for any web property, and that could be verified even as the client navigates across domains.

Essentially: bootstrapping SRI for an initial page-load, and then trusting the server integrity guarantees from there through all subsequent browsing.

Not individually that useful for one person individually since they can't verify the hashes they receive against anything meaningful, but powerful to allow pairs (or more) of people to confirm that they're receiving the same untampered information.

(cross-domain is tricky, I admit; external content changes over time, but a timestamped assertion that validates some previous version of an external page could still be useful)

Anyway: I won't drag too much in that direction, as I'm getting a better sense of the scoping for this spec.

Reply:
For this kind of thing, you may be interested in [IPFS](https://ipfs.tech/), if you're not already aware of it; as it may be a better fit for content :)